services/worker/src/worker/job_runners/dataset/parquet.py

```diff
@@ -81,19 +82,18 @@ def compute_parquet_response(dataset: str) -> Tuple[DatasetParquetResponse, floa
                     )
                 )
                 continue
-            config_parquet_content = ConfigParquetResponse(parquet_files=response["content"]["parquet_files"])
+            config_parquet_content = ConfigParquetResponse(
+                parquet_files=response["content"]["parquet_files"], partial=False
```

Comment:
hmmm isn't it

```suggestion
                parquet_files=response["content"]["parquet_files"], partial=response["content"]["partial"]
```

?